src/helper/util.ts

```diff
@@ -495,6 +523,48 @@ export class ZkWasmUtil {
     console.log("sibling_instances: ", [sibling_instances]);
     console.log("target_instances: ", target_instances);
 
+    let result = await batch_verifier_contract.verify.send(
```

Comment:
Line 523 log change to console.log("sibling_instances: ", sibling_instances); as we now send sibling_instances